lib/stats-collector.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * Stats Collector
+ *
+ * Provides stats such as test duration,
+ * number of tests passed / failed etc.
+ *
+ * @public
```

Comment:
Is this being exported somewhere? Otherwise, should be `@private`

Reply:
I assumed this was flagging the module/function itself as public (is that right?). Its exported and imported by base reporter.

Reply:
Sorry I find this super confusing, the function itself is exported though (i.e. `module.exports = createStatsCollector`). Its publicly available for anything else to import from anywhere.
Do you have an example perhaps of something similar which is private?

Reply:
Mea culpa. I was essentially documenting `@api private` rather than `@public`. Sorry for the confusion.